scripts/seal_tests/tests/2pc_boolean_and_c_bytecode.txt

```diff
@@ -0,0 +1,4 @@
+2 1 a 1 1 IN
```

Comment:
We should run `python3 driver.py --clean` before pushing, so the bytecode shouldn't be included in the commit.